### PR TITLE
Release 4.10.0 🎉

### PR DESCRIPTION
**Pull request by etman55**
## Release Notes

# New
* The `CardComponentState.binValue` now reports 8 digit bins in case of card numbers with 16 or more digits.
* We added `CardBrand`, along with `CardType`, to use when adding new card brands through `CardConfiguration`.
* In `CardConfiguration` when adding supported card types you can now add new supported card types which are not part of `CardType`.
  example:

```kotlin
CardConfiguration.Builder([SHOPPER_LOCALE], [ENVIRONMENT], [CLIENT_KEY])
.setSupportedCardTypes(CardBrand(txVariant = "[CARD_BRAND1]"), CardBrand(txVariant = "[CARD_BRAND2]"))
.build()
```

## Fixed
* Undefined supported card types now works correctly for card component/drop-in.
  `CardType.UNKNOWN` was not working correctly when there were supported card brands coming from `/paymentMethods`, under `scheme` payment method,
  which are not part of `CardType` predefined brands, so we had to add `CardBrand` to make these undefined supported card types work.

## Deprecated
* The `CardType.UNKNOWN` property. Use `CardBrand(txVariant = "[CARD_BRAND]")` instead.
* The `CardType.setTxVariant()` method. No longer needed as it was used with `CardType.UNKNOWN`

### CODE REVIEW

**Comment by jreij** [comment]
Is the card type fix the only thing we're releasing?

**Comment by etman55** [comment]
Yes, AFAIK.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -26,4 +27,4 @@ CardConfiguration.Builder([SHOPPER_LOCALE], [ENVIRONMENT], [CLIENT_KEY])
 
 ## Deprecated
 * The `CardType.UNKNOWN` property. Use `CardBrand(txVariant = "[CARD_BRAND]")` instead.
-* The `CardType.setTxVariant()` method. No longer needed as it was used with `CardType.UNKNOWN`
\ No newline at end of file
+* The `CardType.setTxVariant()` method. No longer needed as it was used with `CardType.UNKNOWN`
```
Comment (jreij) [inline RELEASE_NOTES.md:30]: Missing`.` at the end

File: build.gradle [hunk 2]
```diff
@@ -45,7 +45,7 @@ allprojects {
     // just for example app, don't need to increment
     ext.version_code = 1
     // The version_name format is "major.minor.patch(-(alpha|beta|rc)[0-9]{2}){0,1}" (e.g. 3.0.0, 3.1.1-alpha04 or 3.1.4-rc01 etc).
-    ext.version_name = "4.9.1"
+    ext.version_name = "4.9.2"
```
Comment (jreij) [inline build.gradle:48]: We should make this `4.10.0` since we introduce new functionality

**Comment by jreij** [comment]
The branch is still named `release/4.9.2`, I think we should rename it

**Pull request closed without merging**